share/revng/cmake/TupleTreeGenerator.cmake

```diff
@@ -383,10 +390,15 @@ function(
     list(APPEND SCALAR_TYPE_ARGS --scalar-type "'${ET}'")
   endforeach()
 
+  # We add the `mixins` paramter only when processing the model
+  if(${NAMESPACE} STREQUAL "model")
+    set(MIXINS_PATH_ARG --mixins "${MIXINS_PATH}")
+  endif()
+
   add_custom_command(
     COMMAND
       "${SCRIPTS_ROOT_DIR}/tuple-tree-generate-python.py" --namespace
-      "${NAMESPACE}" --root-type "${ROOT_TYPE}" --output "${OUTPUT_PATH}"
+      "${NAMESPACE}" --root-type "${ROOT_TYPE}" --output "${OUTPUT_PATH}" ${MIXINS_PATH_ARG}
```

Comment:
There should be a `--mixins` here

Reply:
I add the cli args in the loop above, similar to `SCALAR_TYPE_ARGS`, is that ok?

Reply:
Yes